src/renderer/library/components/dialog/Information.tsx

```diff
@@ -34,45 +39,107 @@ interface IProps extends IBaseProps, ReturnType<typeof mapDispatchToProps>, Retu
 }
 
 export class Information extends React.Component<IProps, undefined> {
-    private parsedMarkdown: string;
+
+    private manifestView: PublicationView;
 
     constructor(props: IProps) {
         super(props);
     }
 
     public async componentDidMount() {
-        const { locale } = this.props;
-        const infoFolderRelativePath = "assets/md/information";
+        const { locale, __ } = this.props;
+        const infoFolderRelativePath = "assets/md/information/"; // final / is important
+        const imagesFolder = "images";
+
+        let aboutLocale = locale;
 
-        let folderPath: string = path.join((global as any).__dirname, infoFolderRelativePath);
         try {
+
+            let title = `_______________ ${aboutLocale}`;
+            let [pubView] = await apiAction("publication/search", title);
+            if (pubView) {
+
+                console.log("pubView already exist no need to generate a new one");
+                console.log(pubView);
+
+                this.manifestView = pubView;
+                return;
+            }
+
+            let folderPath = path.join((global as any).__dirname, infoFolderRelativePath);
             if (_PACKAGING === "0") {
                 folderPath = path.join(process.cwd(), "dist", infoFolderRelativePath);
             }
-            const fileContent = await promisify(readFile)(path.join(folderPath,
-                `${locale.toLowerCase()}.md`), {encoding: "utf8"});
-            this.parsedMarkdown = (new commonmark.HtmlRenderer()).render((new commonmark.Parser()).parse(fileContent));
-        } catch (__) {
-            this.parsedMarkdown = "<h1>There is no information for your language</h1>";
+            const filePath = path.join(folderPath, `${aboutLocale}.html`);
+            let htmlFile = `${aboutLocale}.html`;
+            aboutLocale = existsSync(filePath) === true ? locale : "en";
+            htmlFile = `${aboutLocale}.html`;
+            title = `_______________ ${aboutLocale}`;
```

Comment:
A couple of issues:

1) the special `_______________ ` prefix is "declared" twice in the code, I would suggest factoring it out to avoid future problems during code maintenance.
2) Unless I am mistaken:
```
let title = `_______________ ${aboutLocale}`;
let [pubView] = await apiAction("publication/search", title);
```
...never returns a `pubView` because the publication title is set as:
```
publication.Metadata.Title = `${__("reader.footerInfo.moreInfo")} ${aboutLocale}`;
```

Am I missing something?

Reply:
I can see in the console log (command line shell) that the `importFromString` API is called each time, but in the GUI everything looks normal (presumably, because the CRC-based identity prevents importing duplicates? I'm not sure, as there is no "already imported" popup notification)

Reply:
fixes 6046f557